tests/test_crd.py

```diff
@@ -0,0 +1,81 @@
+"""Unit tests for crd.py."""
+
+import os
+import tempfile
+
+import numpy as np
+
+from simSPI import crd
+
+
+def test_write_crd_file():
+    """Test whether function creates and writes to file on given path."""
+    temp_dir = tempfile.TemporaryDirectory()
+    temp_file = temp_dir.name + "/crd.txt"
+
+    assert not os.path.isfile(temp_file)
+    crd.write_crd_file(numpart=20, crd_file=temp_file)
+    assert os.path.isfile(temp_file)
+    crd.write_crd_file(numpart=20, crd_file=temp_file)
+    assert os.path.isfile(temp_file)
```

Comment:
Why is this done twice?

Reply:
tests writing to a file that already exists, should we add a comment to clarify?

Reply:
you could write it in code like 

```
file_already_exists = temp_file
crd.write_crd_file(numpart=20, crd_file=file_already_exists)
```

also, the `assert os.path.isfile(temp_file)` in line 20 would already be written from line 17, and not necessarily 19.
you would want to make a change in the actual file, use the same file name, and check that the new change inside the file that got written to disk is there.